src/Destructurama.Attributed/Attributed/AttributedDestructuringPolicy.cs

```diff
@@ -56,8 +59,29 @@ private static IEnumerable<PropertyInfo> GetPropertiesRecursive(Type type)
             var unseenProperties = type.GetProperties(BindingFlags.Public | BindingFlags.Instance | BindingFlags.DeclaredOnly)
                 .Where(p => p.CanRead && p.GetMethod.IsPublic && p.GetIndexParameters().Length == 0 && !seenNames.Contains(p.Name));
 
+#if NETSTANDARD2_1_OR_GREATER
+            var metaProp = new List<PropertyInfo>();
+            // find Metadata Class
+            // Take only first Entry, unclear from docs whether it can be specified multiple times, would be nonsense from my perspective
+            var metaDataType = type.GetCustomAttributes<MetadataTypeAttribute>(true).ToList().FirstOrDefault();
+            if (metaDataType != null)
+            {
+                var metaClass = metaDataType.MetadataClassType;
+                // find all properties with Custom Attributes which are in referenced class
+                metaProp = metaClass.GetProperties().Where(mp =>mp.CustomAttributes.Count() > 0 && unseenProperties.Any(up => up.Name == mp.Name)).ToList();
+                // replace all found properties in unseenProperties with those from Metadataclass
+                var removedAttr = unseenProperties.Where(up => metaProp.Any(mp => mp.Name != up.Name)).ToList();
+                removedAttr.AddRange(metaProp);
+                unseenProperties = removedAttr;
+            }
+#endif
             foreach (var propertyInfo in unseenProperties)
             {
+#if NETSTANDARD2_1_OR_GREATER
+                if (metaProp.Any(p => p.Name == propertyInfo.Name))
+                {
+                }
+#endif
```

Comment:
?

Reply:
Unclear to me. As long as the library is built for NETSTANDARD 2.0 as well, the check for framework should be there.  Same would be true for associated unit test.

Reply:
This block of code does nothing.

Reply:
Ok, seems to be left over during implementation. I'll remove it.